Functions/calculator.py

```diff
@@ -0,0 +1,28 @@
+first_number = float(input('Enter a number: '))
+operator = input('Enter an operation (+, - , /, *) : ')
+second_number = float(input('Enter a number: '))
+
+def calculate(number1, operator, number2):
+    if operator == '+':
+        return number1 + number2
+    elif operator == '-':
+        return number1 - number2
+    elif operator == '/':
+        return number1 / number2
+    elif operator == '*':
+        return number1 * number2
+    else:
+        return "Invalid operation..."
```

Comment:
I want to see something like this 
https://github.com/MarinaZelenska/test_repo_python/pull/7/files#diff-e38849d79e2ad0d5aea391b4c687352c5b6db1eea0f7c0e6db2286b3078b2314